### DISCUSSION TITLE
How to use live editor

### DISCUSSION THREAD

**Discussion by unicorn64** [retypeapp/retype#349] [Q&A] [answered]
Hi all, Thanks for this great tool.
I couldn't figure out what is "Page content live editor"  and how is it working.
I imagined that i can edit content on browser but i could't find how to.

**Comment by DannyBen** [comment, marked as answer]
When you run `retype watch`, your pages will have this "Edit" button on the top right
![image](https://user-images.githubusercontent.com/2405099/176130240-67a35f74-b42f-45a4-a953-930e45a7c893.png)

**Reply by unicorn64** [reply]
Thanks. Somehow when i have old retype.json config, Edit button not shown. maybe some mistake in config file, I changed to yml and i see
